### PR TITLE
Remove travis changes: Issue 623

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
renamed MONDO_0013308, added text def, revised superclass. Addresses #623

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -263531,23 +263531,27 @@ property_value: confidence "2.0694736842105272" xsd:double
...
+synonym: "Noonan-like syndrome" EXACT []
 synonym: "NSLL" RELATED [MONDO:Lexical, OMIM:613563]
 xref: ICD10:Q87.1 {source="ORDO:363972/attributed", source="ORDO:363972/ntbt", source="Orphanet:363972"}
 xref: OMIM:613563 {source="ORDO:363972/e", source="MONDO:equivalentTo", source="Orphanet:363972"}
 xref: Orphanet:363972 {source="OMIM:613563", source="MONDO:equivalentTo"}
 xref: UMLS:C3150803 {source="OMIM:613563", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
 xref: UMLS:C4016301 {source="MEDGEN:kboom-pr98-c99", source="MONDO:equivalentTo"}
 is_a: MONDO:0015945 {source="Orphanet:363972"} ! polymalformative genetic syndrome with increased risk of developing cancer
-is_a: MONDO:0020297 {source="Orphanet:363972"} ! Noonan syndrome and Noonan-related syndrome
+is_a: MONDO:0021060 {source="ClinGen RASopathy Expert Panel"} ! RASopathy
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: Source should be a CURIE or URI

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: @cmungall what should I use for the source in this case?  There are 14 ORCIDs listed on the ticket for the panel members, should I add all of those ORCIDs?

Comment (cmungall) [reply src/ontology/mondo-edit.obo]: No, not 14!

Is there a URL for the group? If not I suggest setting up one.

But for now, use a GO-style meeting CURIE e.g. ClinGen:RASopathyExpertPanel. No spaces. Let's also collect a doc with full metadata on this organization (but that can be done outside this PR)

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: Thanks - I updated the source. There is not a URL that I know of.
Here is a google doc with the names/ORCIDs for this group: https://docs.google.com/document/d/1t4cj1MZ9xlONOqsScQwbniTprMj83C4Zur1T_na9HMM/edit

Comment (cmungall) [reply src/ontology/mondo-edit.obo]: We can host a URL on mondo site if CLinGen can't

**Comment by nicolevasilevsky** [comment]
@ShahimEssaid can you address this?

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: Ok, I'll ask ClinGen

**Comment by nicolevasilevsky** [comment]
Actually, I can just redo this in a separate PR

**Comment by nicolevasilevsky** [comment]
closing, as I redid these edits in PR https://github.com/monarch-initiative/mondo/pull/776

**Pull request closed without merging**